src/modules/users/components/AvatarDropdown/AvatarDropdownPopover.tsx

```diff
@@ -119,7 +119,7 @@ const AvatarDropdownPopover = ({
     <DropdownMenuSection separator>
       <DropdownMenuItem>
         <ExternalLink
-          href="https://colony.canny.io"
+          href="https://portal.productboard.com/colony/1-colony-portal/tabs/4-bugs"
```

Comment:
This is giving a lint error due to how long the URL is on the line.

You can get around this by setting the URL as a constant. For example:

![long-urls](https://user-images.githubusercontent.com/33682027/144844475-9db86a05-0825-4a41-af4f-afeacbef0098.jpg)